standard/docs/en/getting_started/releases_and_records.md

```diff
@@ -97,7 +97,7 @@ In some cases, releases will all come from the same backend system. In others, t
 
 In a complete OCDS implementation, each release would be published at its own persistent URL, and kept online permanently. 
 
-In these cases, where publishers are providing a single URI for fetching information on a given contracting process, the `releaseID` and `releaseDate` must be modified whenever updated data is published. A suggested pattern is to append an incrementing number to the end of the `ocid` in order to generate a unique `releaseID` for every change. For example:
+In some cases, a publisher might only have the data infrastructure to publish and iteratively update one release for each contracting process. If so, the release `id` and release `date` must be modified with each update. A suggested pattern for generating a unique release `id` is to append an incrementing number to the end of the `ocid`: for example, if the `ocid` is "ocds-a2ef3d01-1594121", then each sequential release `id` can be:
```

Comment:
My understanding is that this isn't always easy to implement depending on how APIs are structured, so we should expand it to include the recommendations we provide in the data feedback report:

> **Considerations for implementation of unique release identifiers**
> If the database record for each contracting process includes a last updated date, this can be appended to the `ocid` each time the data is exported from the database or fetched from an API to produce a unique release identifier. Otherwise a hash of all the fields in the release can be used to generate a unique release identifier.

Reply:
Let's make that part of the 1.1.5 example. I added the above to the spreadsheet.

Reply:
OK, sounds good.